### ISSUE TITLE
multiple GPUs when initialize

### ISSUE THREAD

**Issue by chuong98** [GRAAL-Research/poutyne#11]
Hi, thanks for the package. Does the package support multiple -GPUs mode.
Some thing like this when initializing the Model.__init__()

```
def __init__(self, model, optimizer, loss_function, metrics=[]):
        self.model = model
        self.optimizer = optimizer
        self.loss_function = loss_function
        self.metrics = list(map(get_metric, metrics))
        self.metrics_names = [metric.__name__ for metric in self.metrics]
        self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
        print(f'Using GPU {self.device}')

        #Transfer model to GPUs model
        nGPUs =   torch.cuda.device_count()
        if  nGPUs> 1:
            print(f"==============Using {nGPUs} GPUs ==============")
            self.model = torch.nn.DataParallel(self.model)
        self.model = self.model.to(self.device)

```

**Comment by freud14** [comment]
Hi,
Thank you for your suggestion. I just made a branch with a attempt at supporting multiple GPUs. I try to test it with some transfer learning but the loss was exploding whereas it would not on single GPU. Maybe it has to do with batch normalization? I don't know. Can you test it on your side and tell me if it works fine for you? If so I will merge the branch in the dev branch. 

Thank you.

**Comment by freud14** [comment]
Hi, have you taken the time to test the commit?

**Comment by davebulaval** [comment]
@chuong98 Did you try the branch `multiple-gpus` for multiple GPU ? We would like to close this issue.
tks!
